cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -781,15 +781,16 @@
   public static final String GOAL_BALANCEDNESS_PRIORITY_WEIGHT_CONFIG = "goal.balancedness.priority.weight";
   private static final String GOAL_BALANCEDNESS_PRIORITY_WEIGHT_DOC = "The impact of having one level higher goal priority"
        + " on the relative balancedness score. For example, 1.1 means that a goal with higher priority will have the 1.1x"
-       + " balancedness weight of the lower priority goal (assuming the same goal.balancedness.strictness.weight values).";
+       + " balancedness weight of the lower priority goal (assuming the same goal.balancedness.strictness.weight values for"
+       + " both goals).";
 
   /**
    * <code>goal.balancedness.strictness.weight</code>
    */
   public static final String GOAL_BALANCEDNESS_STRICTNESS_WEIGHT_CONFIG = "goal.balancedness.strictness.weight";
-  private static final String GOAL_BALANCEDNESS_STRICTNESS_WEIGHT_DOC = "The impact of strictness on the relative "
-       + " balancedness score. For example, 1.5 means that a hard goal will have the 1.5x balancedness weight of a soft "
-       + "goal (assuming goal.balancedness.priority.weight is 1).";
+  private static final String GOAL_BALANCEDNESS_STRICTNESS_WEIGHT_DOC = "The impact of strictness (i.e. hard or soft goal)"
+       + " on the relative  balancedness score. For example, 1.5 means that a hard goal will have the 1.5x balancedness "
```

Comment:
nit: redundant white space here(double whitespace)

Reply:
-- went over the code to fix similar issues globally.